directory_sso_api_client/testapiclient.py

```diff
@@ -13,3 +13,10 @@ def get_user_by_email(self, email: str):
             url = self.endpoints['user_by_email'].format(email=email)
             result = self.get(url=url)
         return result
+
+    def delete_user_by_email(self, email: str):
+        result = None
+        if email:
```

Comment:
in a method called `delete_user_by_email`, why would email ever be `None`?

Reply:
It covers situation when `email=None` or `email=""` and saves a redundant request to the sso proxy.